Source/CheckoutSdkIos.swift

```diff
@@ -4,7 +4,7 @@ import UIKit
 /// Add a keyboard toolbar with navigation buttons. Use it to navigate between text fields.
 ///
 /// - parameter textFields: Array of text fields
-public func addKeyboardToolbarNavigation(textFields: [UITextField]) {
+func addKeyboardToolbarNavigation(textFields: [UITextField]) {
```

Comment:
```suggestion
private func addKeyboardToolbarNavigation(textFields: [UITextField]) {
```

Reply:
This function is used across multiple views in Frames. We should keep it non-private for now